src/App.js

```diff
@@ -1,13 +1,72 @@
 import React from 'react';
 import './App.css';
+import Button from "./Components/Button";
+import Product from "./Components/Product";
+import Tile from "./Components/Tile";
+import bag1 from "./assets/bag_1.png"
+import bag2 from "./assets/bag_2.png"
+import bag3 from "./assets/bag_3.png"
+import bag4 from "./assets/bag_4.png"
+
+
 
 function App() {
-  return (
-      <h1>Handbags & Purses</h1>
-  );
+    return (
+        <>
+            <h1>Handbags & Purses</h1>
+            <nav>
+                <Button nameButton="To the collection" disabled={false}/>
+                <Button nameButton="Shop all bags" disabled={false}/>
+                <Button nameButton="Pre-orders" disabled={true}/>
+            </nav>
+            <main>
+                <Product
+                    productTag="Best seller"
+                    srcPath={bag1}
+                    nameProduct="The handy bag"
+                    price="400"/>
+                <Product
+                    productTag="Best seller"
+                    srcPath={bag2}
+                    nameProduct="The stylish bag"
+                    price="250"/>
+                <Product
+                    productTag="New collection"
+                    srcPath={bag3}
+                    nameProduct="The simple bag"
+                    price="300"/>
+                <Product
+                    productTag="New collection"
+                    srcPath={bag4}
+                    nameProduct="The trendy bag"
+                    price="150"/>
+            </main>
+            <footer>
+                <Tile>
+                    <h2>The brand</h2>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Beatae, inventore suscipit. Aliquid asperiores perspiciatis tenetur.</p>
+                    <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Asperiores ducimus itaque nemo nisi, nulla quasi?</p>
+                </Tile>
+                <Tile>
+                    <img src={require("./assets/brand.png").default} alt="brand logo"/>
```

Comment:
Ik kan deze code niet goed lezen, stel je hiermee een default in? Het antwoord model hanteert de volgende schrijfwijze: <Tile img={brand} imgDescription="The brand logo" />

Reply:
Ik had problemen met het importeren van de img files in bij app.js. Door de googlen kwam ik erachter dat je inplaats van importeren van je img het ook op deze manier kan schrijven. Ik heb echter geen idee waarom je er default achter moet zetten. 

Ik mist uiteindelijk een ./ aan het begin van paths waardoor het importeren niet ging 🙃